[2024-02-10 06:28:24,438] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: example_bash_operator.also_run_this scheduled__2024-02-09T00:00:00+00:00 [queued]>
[2024-02-10 06:28:24,452] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: example_bash_operator.also_run_this scheduled__2024-02-09T00:00:00+00:00 [queued]>
[2024-02-10 06:28:24,452] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-02-10 06:28:24,452] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2024-02-10 06:28:24,452] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-02-10 06:28:24,491] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): also_run_this> on 2024-02-09 00:00:00+00:00
[2024-02-10 06:28:24,503] {standard_task_runner.py:52} INFO - Started process 29859 to run task
[2024-02-10 06:28:24,519] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'example_bash_operator', 'also_run_this', 'scheduled__2024-02-09T00:00:00+00:00', '--job-id', '6', '--raw', '--subdir', '/home/***/.local/lib/python3.7/site-packages/***/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmp4dv9egq_', '--error-file', '/tmp/tmpy1crph7q']
[2024-02-10 06:28:24,520] {standard_task_runner.py:80} INFO - Job 6: Subtask also_run_this
[2024-02-10 06:28:24,949] {task_command.py:369} INFO - Running <TaskInstance: example_bash_operator.also_run_this scheduled__2024-02-09T00:00:00+00:00 [running]> on host f4c5b03650dc
[2024-02-10 06:28:25,520] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=also_run_this
AIRFLOW_CTX_EXECUTION_DATE=2024-02-09T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-02-09T00:00:00+00:00
[2024-02-10 06:28:25,530] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2024-02-10 06:28:25,536] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "run_id=scheduled__2024-02-09T00:00:00+00:00 | dag_run=<DagRun example_bash_operator @ 2024-02-09 00:00:00+00:00: scheduled__2024-02-09T00:00:00+00:00, externally triggered: False>"']
[2024-02-10 06:28:25,570] {subprocess.py:85} INFO - Output:
[2024-02-10 06:28:25,589] {subprocess.py:92} INFO - run_id=scheduled__2024-02-09T00:00:00+00:00 | dag_run=<DagRun example_bash_operator @ 2024-02-09 00:00:00+00:00: scheduled__2024-02-09T00:00:00+00:00, externally triggered: False>
[2024-02-10 06:28:25,591] {subprocess.py:96} INFO - Command exited with return code 0
[2024-02-10 06:28:25,711] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=also_run_this, execution_date=20240209T000000, start_date=20240210T062824, end_date=20240210T062825
[2024-02-10 06:28:25,850] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-02-10 06:28:25,945] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
